canonical-web-and-design/ubuntu.com
# Rewrite advantage to use talisker session

This is in preparation for my work with the image builder
API, as they should both be using the shared talisker.session.

This led me to restructure the login.py and advantage API code a bit.

I've also slightly tidied up the tutorials index view by moving it
into views.py.

QA
--

Go to `/advantage`, check login still works as expected. Try logging and and logging out and viewing your UA token etc.
